pom.xml

```diff
@@ -260,10 +260,11 @@
                 <plugin>
                     <groupId>org.eclipse.jetty</groupId>
                     <artifactId>jetty-maven-plugin</artifactId>
-                    <version>9.3.0.M1</version>
+                    <version>9.2.21.v20170120</version>
```

Comment:
This is counterintuitive but >9.3 upgraded to Java 8 and point releases didn't run on Java 7. Better to be on the latest version of the last Java 7 series.